florestad/src/florestad.rs

```diff
@@ -489,6 +492,37 @@ impl Florestad {
             .map(|addr| Self::get_ip_address(&addr, 50001))
             .unwrap_or("0.0.0.0:50001".parse().expect("Hardcoded address"));
 
+        // generate self-signed certificate if provided
+        if self.config.gen_selfsigned_cert {
+            // create ssl dir if not exists
+            let ssl_dir = format!("{}ssl", &data_dir);
+            if !std::path::Path::new(&ssl_dir).exists() {
```

Comment:
Could you avoid using those full paths here? Import the type you need and call it directly. e.g.: `Path::new` instead of `std::path::new`. They only exceptions here are the errors that are literally called `Error` and maybe the methods in `fs` like `fs::write`.